### ISSUE TITLE
Migration and Seeds on Production error: 

### ISSUE THREAD

**Issue by anyav77** [susanBuck/e15-spring20#61]
I tried deploying my Bookmark work in progress on production, and I got an error:
```git
  SQLSTATE[HY000]: General error: 1364 Field 'author' doesn't have a default value (SQL: insert into `books` (`slug`, `title`, `published_year`
  , `cover_url`, `info_url`, `purchase_url`, `description`, `updated_at`, `created_at`) values (the-martian, The Martian, 2011, https://hes-boo
  kmark.s3.amazonaws.com/the-martian.jpg, https://en.wikipedia.org/wiki/The_Martian_(Weir_novel), https://www.barnesandnoble.com/w/the-martian-
  andy-weir/1114993828, The Martian is a 2011 science fiction novel written by Andy Weir. It was his debut novel under his own name. It was ori
  ginally self-published in 2011; Crown Publishing purchased the rights and re-released it in 2014. The story follows an American astronaut, Ma
  rk Watney, as he becomes stranded alone on Mars in the year 2035 and must improvise in order to survive., 2020-05-09 03:55:55, 2020-05-09 03:
  55:55))
```

![migration-seeds-production-error](https://user-images.githubusercontent.com/1045307/81463911-27e77180-9172-11ea-89ca-f69685561e2e.png)

**Comment by susanBuck** [comment]
Hi @anyav77 - FYI I'm cloning your project locally now to test.

**Comment by susanBuck** [comment]
By project, I meant repository. : )

I tested your `bookmark` app locally and seeds/migrations ran without error. I assumed this would be the case because judging from your post, I'm thinking you're not having this issue locally either - it's isolated to production. Still - just wanted to double check.

Is it possible not all changes you've made have been sync'd with production yet? The reason I ask is because the error is referring to an `author` field, which is something we used to have in bookmark but have since replaced with `author_id`.
